test/e2e/test/apmserver/checks_apm.go

```diff
@@ -264,3 +267,53 @@ func countIndex(esClient client.Client, indexName string) (int, error) {
 	}
 	return countResult.Count, nil
 }
+
+const sampleDefaultAgentConfiguration = `{"service":{},"settings":{"transaction_sample_rate":"1","capture_body":"errors","transaction_max_spans":"99"}}`
+
+// CheckAgentConfiguration creates an agent configuration through Kibana and then check that the APM Server is able to retrieve it.
+func (c *apmClusterChecks) CheckAgentConfiguration(apm apmv1.ApmServer, k *test.K8sClient) test.StepList {
+	apmVersion := version.MustParse(apm.Spec.Version)
+
+	if !(apmVersion.IsSameOrAfter(apmv1.ApmAgentConfigurationMinVersion) && apm.Spec.KibanaRef.IsDefined()) {
+		return []test.Step{}
+	}
+
+	return []test.Step{
+		{
+			Name: "Create the default Agent Configuration in Kibana",
+			Test: func(t *testing.T) {
+				kb := kbv1.Kibana{}
+				assert.NoError(t, k.Client.Get(apm.Spec.KibanaRef.WithDefaultNamespace(apm.Namespace).NamespacedName(), &kb))
+
+				password, err := k.GetElasticPassword(apm.Spec.ElasticsearchRef.WithDefaultNamespace(apm.Namespace).NamespacedName())
+				assert.NoError(t, err)
+
+				uri := "/api/apm/settings/agent-configuration"
+
+				// URI is slightly different before 7.7.0, we need to add "/new" at the end
+				if !apmVersion.IsSameOrAfter(version.MustParse("7.7.0")) {
+					uri += "/new"
```

Comment:
I understand we add the `/new` prefix when the version of the APM Server is equal or greater than 7.7.0. Is it not the opposite that we want?